Lib/fontTools/feaLib/ast.py

```diff
@@ -77,6 +81,15 @@ def deviceToString(device):
         return "<device %s>" % ", ".join("%d %d" % t for t in device)
 
 
+def statNameToString(name_rec):
+    platformID, platEncID, langID, string = name_rec
+    if platformID == 3 and platEncID == 1 and langID == 0x0409:
+        return f'name "{string}";'
+    string = string.encode('unicode-escape').decode().replace('\\u', '\\').upper()
+    langID = '0x%04x' % langID
+    return f'name {platformID} {platEncID} {langID} "{string}";'
```

Comment:
There are many name entries elsewhere in this module (`NameRecord`, `FeatureNameStatement`, `CVParametersNameStatement`) and they don’t do any escaping. I suggest you try to match their code for now, and open a separate issue if you think it should do escaping. It is better to be consistent here, IMO.

Reply:
agreed

Reply:
Done. Dropped all this and used FeatureNameStatement.